wgsl/index.bs

```diff
@@ -348,6 +348,19 @@ literal_or_ident
     Specifies the binding group of the resource.
     See [[#resource-interface]].
 
+  <tr><td><dfn noexport dfn-for="attribute">`interpolate`
+    <td>One or two parameters.
```

Comment:
Could we consider having them as separate attributes?
We had a similar choice for bind groups, ending up with separate `binding()` and `group()`, on the basis that there are already attributes that are interlinked (e.g. `stage(compute)` has to be used with `workgroup_size(...)`).
So having two separate attributes here may end up being more consistent with the rest of the spec.

Reply:
I guess this raises another point:
- if perspective or linear is used, then the second parameter must be supplied. (No defaulting?)

That should be stated

I think having two different attributes does not gain anything and instead only opens up new opportunities for error.